styles.css

```
@@ -19,15 +22,20 @@
   border: 5px;
   border-radius: 5px;
   border-color: rgb(73, 10, 51);
-  width: 50%;
+  width: 100%;
+  height: 80px;
   text-align: center;
+  text-wrap: balance;
+}
+
+.fa-10x {
+  align-content: flex-end;
 }
 
 .bookmark {
-  position: fixed;
+  position: flex;
   height: 70px;
   width: 50px;
-  padding: 0px;
   box-sizing: border-box;
```

This rule should apply to the universal selector and should not be included within your `.bookmark` class.
```suggestion
* {
  box-sizing: border-box;
  }
```